src/lib/components/home/HomeRow.svelte

```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
```

Comment:
Naming: This is not really a row, right? It's just one element in the row.

Reply:
yup good catch will rename